OpenSim/Actuators/MuscleFirstOrderActivationDynamicModel.h

```diff
@@ -116,38 +110,6 @@ OpenSim_DECLARE_CONCRETE_OBJECT(MuscleFirstOrderActivationDynamicModel, Object);
                                            double minActivation,
                                            const std::string& muscleName);
 
-    void ensureModelUpToDate();
-
-    /** @returns The activation time constant (in seconds). */
-    double getActivationTimeConstant() const;
-
-    /** @returns The deactivation time constant (in seconds). */
-    double getDeactivationTimeConstant() const;
-
-    /** @returns The lower bound on activation. */
-    double getMinimumActivation() const;
-
-    /** @returns The upper bound on activation. */
-    double getMaximumActivation() const;
-
-    /**
-    @param activationTimeConstant The activation time constant (in seconds).
-    @returns A boolean indicating whether the value was set.
-    */
-    bool setActivationTimeConstant(double activationTimeConstant);
-
-    /**
-    @param deactivationTimeConstant The deactivation time constant (in seconds).
-    @returns A boolean indicating whether the value was set.
-    */
-    bool setDeactivationTimeConstant(double deactivationTimeConstant);
-        
-    /**
-    @param minimumActivation The lower bound on activation.
-    @returns A boolean indicating whether the value was set.
-    */
-    bool setMinimumActivation(double minimumActivation);
-
     /**
     @returns Activation clamped to the range [minActivation, 1.0].
     */
```

Comment:
This issue will be addressed once MuscleFirstOrderActivationDynamicModel has been replaced with FirstOrderMuscleActivationDynamics, which owns the activation state. See [FirstOrderMuscleActivationDynamics::getActivation()](https://github.com/opensim-org/opensim-core/blob/master/OpenSim/Actuators/FirstOrderMuscleActivationDynamics.cpp#L111).